### PR TITLE
[PBW-4554] Fix for buffer wheel appearing throughout main video playback

### PR DESCRIPTION
**Pull request by nagavenivn**

### CODE REVIEW

**Comment by nagavenivn** [comment]
@chibzu/ @mityaha could any one of you please review this pull request?

File: src/main/js/main_html5.js [hunk 1]
```diff
@@ -872,6 +872,10 @@ require("../../../html5-common/js/utils/environment.js");
      * @method OoyalaVideoWrapper#raiseWaitingEvent
      */
     var raiseWaitingEvent = _.bind(function() {
+      // WAITING event is not raised if no video is assigned yet
+      if (_video.currentSrc === "" || _video.currentSrc === null) {
```
Comment (mityaha) [inline src/main/js/main_html5.js]: I recommend to use if (_.isEmpty(_video.currentSrc)) { - it covers all the cases

**Comment by mityaha** [comment]
Is it feasible to add unit test for this case?

File: src/main/js/main_html5.js (hunk 1 above)
Comment (nagavenivn) [inline src/main/js/main_html5.js]: @mityaha : Modified if condition as per your comment

Comment (nagavenivn) [inline src/main/js/main_html5.js]: @mityaha  Also added new test case and modified existing test case to work with the fix

**Comment by mityaha** [comment]
Looks good!

**Pull request merged**